articlesView.js

```diff
@@ -0,0 +1,60 @@
+class ArticlesView {
+  constructor(model, api) {
+    this.model = model;
+    this.api = api;
+    this.mainContainerEl = document.querySelector('#main-container');
+
+    document.querySelector('#search-content-btn').addEventListener('click', () => {
+      this.model.reset();
+      
+      const query = document.querySelector('#search-content-input').value;
+      console.log('Query: ', query);
+
+      this.api.loadArticles((articles) => {
+        let articlesList = articles.response.results;
+        console.log(articlesList);    // Log list of articles
+        this.model.setArticles(articlesList);
+        this.displayArticles();
+        // location.reload();
+      }, query);
+    });
+  }
+
+  displayArticles() {
```

Comment:
Creating a few private functions to break this function down would help with the SRP

Reply:
Thank you! This is really good advice and I noticed you did this very well in your project which was really great to see.